hunting/README.md

```diff
@@ -0,0 +1,60 @@
+# Hunt Queries
+
+---
+
+Welcome to the `hunting` folder within the `detection-rules` repository! This directory houses a curated collection of threat hunting queries designed to enhance security monitoring and threat detection capabilities using the Elastic Stack. Each file in this directory provides a query tailored for identifying specific security threats or suspicious activities.
+
+These queries are designed for use with the Elastic Security platform, part of the broader Elastic Stack, enabling security teams to proactively hunt for potential threats in their environment.
+
+- KQL
+- EQL
+- ES|QL
+- OsQuery
+- YARA
+
+## How to Contribute
+
+Contributing to the `hunting` folder is a great way to share your expertise and enhance the security community's capabilities. Here’s how you can contribute:
+
+### Adding New Queries
+- **TOML File Naming and Organization**: Ensure that any new queries are named descriptively and grouped by the type of threat they address. Place your TOML files inside the `queries` folder and ensure they are named in a way that reflects the nature of the threat or behavior they are designed to detect.
+- **TOML Fields**: To ensure the hunt queries are consistent and comprehensive, it's important to structure the threat detection rules with specific fields. When contributing a new rule, please include the following fields in the TOML file to describe and configure the analytic:
+  - **author**: The name of the individual or organization authoring the rule.
+  - **datasource**: The specific integration or data source the rule applies to, such as `aws_bedrock.invocation`.
```

Comment:
Maybe we add some additional language here? 

Data sources are based on Elastic integrations typically. These often will resemble the value in ECS field event.dataset

??